minecraft/minecraft-test/build.gradle

```diff
@@ -35,6 +36,7 @@ dependencies {
 		exclude module: 'launchwrapper'
 		exclude module: 'guava'
 	}
+	annotationProcessor "io.github.llamalad7:mixinextras-common:$mixin_extras_version"
```

Comment:
Question: Should this be mixinextras-fabric? to match what will be used in normal development envs?